content/en/docs/instrumentation/js/getting-started/nodejs.md

```diff
@@ -306,6 +315,59 @@ few times, after a while you should see the spans printed in the console by the
   "status": { "code": 1 },
   "events": []
 }
+```
+
+</details>
+
+The generated span tracks the lifetime of a request to the `/rolldice` route.
+
+Send a few more requests to the endpoint. After a moment, you'll see metrics in
+the console output, such as the following:
+
+<details>
+<summary>View example output</summary>
+
+```jsonlines
```

Comment:
The block lang `jsonlines` results in no highlighting. We get best results with `yaml`:

```suggestion
```yaml
```